pyop2/sequential.py

```diff
@@ -441,6 +446,53 @@ def c_map_init(self, is_top=False, is_facet=False):
                                     'off': ' + ' + str(m.offset[idx])})
         return '\n'.join(val)+'\n'
 
+    def c_map_bcs_variable(self, sign, is_facet):
+        if is_facet:
+            raise NotImplementedError
```

Comment:
The horizontal facet case needs to glue together masks manually.

The vertical interior facet case also needs a change, because I don't store one mask for the interior facet map but a pair (for the left and right halves).